### ISSUE TITLE
DOC: regenerate examples and small fixes

### ISSUE THREAD

**Issue by bocklund** [pycalphad/pycalphad#321]
Examples could be regenerated and some low hanging fruit issues could be tackled (few lines of changes maximum for each item):

- [ ] Add `v.N: 1` as a condition for all `equilibrium` calls and `N=1` for all `calculate` calls.
- [ ] Make sure xarray output is using the newer rich HTML output, which is what users will see
- [ ] Binary phase diagram examples: when creating a figure in the notebook, use a higher DPI (maybe 150 or 200?) instead of increasing the `figsize`
- [ ] "Calculating Energy Surfaces of Binary Systems" - do the calculate step for all phases in one call and use xarray `calc_res.where(calc_res.Phase == phase_name)` syntax for masking in the loop
- [ ] "NP vs. T" - use the broadcast_like functionality of xarray, similar to the cookbook example for "NP vs. T" [here] 
- [ ] Prefer xarray indexing and broadcasting to numpy indexing wherever possible (see, for example the ["NP vs. T" cookbook example](https://github.com/pycalphad/cookbook/blob/master/PhaseCompositions/PhaseCompositions.ipynb))
- [ ] Change the reference state of the convex hull enthalpy of formation example to be at 298.15 K.
- [ ] Activity example: Use `v.R` instead of hardcoded 8.315
- [ ] Activity example: change the title of the plot to "1023 K" to match the data

Related: it would be good to be running the docs notebooks as part of our CI to catch any docs regressions. The website build could regenerate the docs on-the-fly in each build, but it probably isn't necessary.

**Comment by disouzam** [comment]
Hello, @bocklund :

I'm new to xarray and I didn't search yet all the places where it is used in pycalphad. 

Looking at the second item of this issue (Make sure xarray output is using the newer rich HTML output, which is what users will see), do you mean the output shown in the picture below (taken from examples I'm following from PyData Prague 2020 talk using marimo's notebook)?:

<img width="1174" height="627" alt="Image" src="https://github.com/user-attachments/assets/ff94cbcd-d6c8-4e57-a904-81b37df92b6d" />

**Comment by bocklund** [comment]
No, it's slightly different. For example, in the "Exploring `calculate` and `equilibrium` xarray Datasets" notebook, changing `print(calc_result)` to just `calc_result` in the first calculation cell gives this interactive HTML element 

<img width="879" height="408" alt="Image" src="https://github.com/user-attachments/assets/f0ae7474-7ddc-46fc-b420-46344e298278" />

that can be interacted with to, for example, expand out the data

<img width="872" height="753" alt="Image" src="https://github.com/user-attachments/assets/0cfb0115-17c6-40f0-a6f0-0b7856b30d28" />

the idea is to get this rich output to go through the nbconvert so it gets into the docs

**Comment by disouzam** [comment]
Thanks for the clarification, @bocklund ! I will explore "Exploring calculate and equilibrium xarray Datasets" notebook to get familiar with it.

**Comment by disouzam** [comment]
@bocklund :

I guess this is the expected result, right?

This is from the documentation, built and served locally, with the corresponding changes in the notebook [UsingCalculationResults.ipynb](https://github.com/pycalphad/pycalphad/blob/develop/examples/UsingCalculationResults.ipynb)

<img width="863" height="753" alt="Image" src="https://github.com/user-attachments/assets/f39286dd-4907-47d1-a75e-2093fc76e471" />

**Comment by bocklund** [comment]
Looks great! Not super important to getting these merged in, but I'm curious how you installed PyCalphad since it seems like the version is not getting detected correctly under the attributes.

**Comment by disouzam** [comment]
Thanks! I started exploring pycalphad last year when you weren't using uv. So, to simplify my virtual environment creation and setup, I added a [custom bash script](https://github.com/disouzam/pycalphad/blob/learning-activities-2026/install_development_version_windows.sh) for my own use and to differentiate between Windows and WSL, I have added a suffix to virtual environment name (which is the default environment in my VS Code). I guess this is the source of the "version error".

.venv environment created by uv:

<img width="695" height="260" alt="Image" src="https://github.com/user-attachments/assets/5b580f97-516f-44a0-a518-1d17a4fbb44c" />

My custom .venv-win environment:

<img width="628" height="321" alt="Image" src="https://github.com/user-attachments/assets/595f80ac-0382-483c-89d3-9afd0968329d" />

The docs, however, appear to pick the latest git commit on which I'm running build process:

<img width="1094" height="792" alt="Image" src="https://github.com/user-attachments/assets/9814f5ca-0788-4189-b9ea-cd4f4b7166fb" />

**Comment by disouzam** [comment]
Hi, @bocklund : #668 is ready to review. I assumed there is no other example that provided `xarray` output but I will do a double check now.
